examples/language-modeling/run_lora_clm.py

```diff
@@ -111,6 +113,15 @@ class ModelArguments:
             "help": "should enable when using custom model architecture that is not yet part of the Hugging Face transformers package like MPT)."
         },
     )
+    low_cpu_mem_usage: bool = field(
+        default=False,
+        metadata={
+            "help": (
+                "It is an option to create the model as an empty shell, then only materialize its parameters when the pretrained weights are loaded."
+                "set True will benefit LLM loading time and RAM consumption."
```

Comment:
```suggestion
                "It is an option to create the model as an empty shell, then only materialize its parameters when the pretrained weights are loaded. "
                "When set to `True`, it will benefit LLM loading time and RAM consumption."
```